server/aws-lsp-codewhisperer/src/language-server/telemetry/userTriggerDecision.test.ts

```diff
@@ -760,15 +763,15 @@ describe('Telemetry', () => {
                     aUserTriggerDecision({
                         codewhispererSessionId: 'cwspr-session-id-0',
                         codewhispererSuggestionState: 'Discard',
-                        codewhispererTimeToFirstRecommendation: 2520,
+                        codewhispererTimeToFirstRecommendation: 1260,
```

Comment:
was this an intentional change?

Reply:
Yes. This is caused by changing nested callback in CodeWhispererServer to `async` and I suspect it changes the chain of async callbacks resolution. The test emulates several inflight requests, and only this time in test environment is impacted. 
In my manual E2E testing with compiled server, times are at same rate as before this change. I didn't want to refactor this test as it too complex :(